File: github/u00-nivelamento/02-exercicios/ex3.c
#include <stdio.h>
#include <stdlib.h>

void extremos(int* arr, int n) {

    int menor = arr[0], maior = arr[0];

    for(int i = 1; i < n; i++) {
        if(arr[i] < menor) menor = arr[i];
        if(arr[i] > maior) maior = arr[i];
    }

    printf("Menor: %d\nMaior: %d\n", menor, maior);
}

int main(void) {

    int arr[] = {5,1,2,9,7,6};
    int n = sizeof(arr) / sizeof(arr[0]);

    extremos(arr, n);

    return 0;
}

File: b/github/u00-nivelamento/02-exercicios/ex3.java
class ex3 {

    public static void extremos(int[] arr) {
        int menor = arr[0], maior = arr[0];

        for(int i = 1; i < arr.length; i++) {
            if(arr[i] < menor) menor = arr[i];
            if(arr[i] > maior) maior = arr[i];
        }

        System.out.println("Menor: " + menor + "\nMaior: " + maior);
    }

    public static void main(String[] args) {

        int[] arr = new int[]{5,1,2,9,7,6};

        extremos(arr);
    }

}